api/src/pcapi/scripts/matching_acceslibre/main.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from math import ceil
+
+import click
+import sqlalchemy as sa
+
+from pcapi.core.offerers import api as offerers_api
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+from pcapi.utils.blueprint import Blueprint
+
+
+blueprint = Blueprint(__name__, __name__)
+logger = logging.getLogger(__name__)
+
+BATCH_SIZE = 1000
+
+
+@blueprint.cli.command("find_new_match_at_acceslibre")
+@click.option("--dry-run", type=bool, default=True)
+def find_new_match_at_acceslibre(dry_run: bool = False, start_from_batch: int = 1) -> None:
+    venues_query = (
+        offerers_models.Venue.query.outerjoin(offerers_models.Venue.accessibilityProvider)
+        .filter(
+            offerers_models.Venue.isPermanent == True,
+            offerers_models.Venue.isVirtual == False,
+            offerers_models.AccessibilityProvider.id.is_(None),
+        )
+        .options(
+            sa.orm.load_only(
+                offerers_models.Venue.name,
+                offerers_models.Venue.publicName,
+                offerers_models.Venue.street,
+                offerers_models.Venue.banId,
+                offerers_models.Venue.siret,
+            )
+        )
+    )
+
+    num_batches = ceil(venues_query.count() / BATCH_SIZE)
+    venues_without_acceslibre = venues_query.all()
```

Comment:
Ici, tu fais deux requêtes, la première pour un `SELECT COUNT`, puis la seconde pour `SELECT name, "publicName", street ...`. Je ne vois pas trop le besoin du `count()` alors que tu fais directement un `all()` derrière.
Tu gagnerais une requête ainsi : 
```suggestion
    venues_without_acceslibre = venues_query.all()
    num_batches = ceil(len(venues_without_acceslibre) / BATCH_SIZE)
```
À moins que j'aie raté quelque chose dans ma compréhension ?